app/controllers/users_controller.rb

```diff
@@ -48,4 +49,8 @@ def set_user
     @user = User.find_by_id(user_id)
   end
 
+  def set_owner
+    @owner = @user
```

Comment:
This is a bit of a "misdirection", especially given that the `@owner` reference is then used all the way over in the SessionHelper